pay-api/src/pay_api/models/statement_settings.py

```diff
@@ -40,10 +40,18 @@ def find_active_settings(cls, account_id: str, valid_date: datetime):
         """Return active statement setting for the account."""
         valid_date = get_local_time(valid_date)
         query = cls.query.join(PaymentAccount).filter(PaymentAccount.auth_account_id == account_id)
+        # need this to strip of the time information from the date
+        todays_datetime = datetime(valid_date.today().year, valid_date.today().month, valid_date.today().day)
```

Comment:
datetime date() function won't work here?

Reply:
I will check/fix this in my next PR.. need some tests to verify it...
the problem was the timestamp information was messing up ; so needed something with just date format in dateitime...